cmd/notary/keys.go

```diff
@@ -424,6 +425,19 @@ func (k *keyCommander) keysRotate(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+
+	if rotateKeyRole == data.CanonicalRootRole {
+		cmd.Println("Warning: you are about to rotate your root key.\n" +
+			"You will still need your old key to sign future root changes\n" +
+			"so that clients who may not be behind can update.\n" +
```

Comment:
This doesn't read right: "clients who may not be behind"

Not sure what the right wording is as it doesn't matter how up to date a client is, when a root rotation happens, if the old key isn't used to sign, everyone has to manually reset.